discovery-infra/day2.py

```diff
@@ -290,3 +304,20 @@ def set_cluster_proxy(client, cluster_id, copy_proxy_from_cluster, args):
     https_proxy = args.https_proxy if args.https_proxy else copy_proxy_from_cluster.https_proxy
     no_proxy = args.no_proxy if args.no_proxy else copy_proxy_from_cluster.no_proxy
     client.set_cluster_proxy(cluster_id, http_proxy, https_proxy, no_proxy)
+
+def set_nodes_hostnames_if_needed(client,
+                                  terraform_cluster_dir_prefix,
+                                  namespace,
+                                  with_static_network_config,
+                                  has_ipv_4,
+                                  network_name,
+                                  cluster_id):
+    tf_folder = utils.get_tf_folder(terraform_cluster_dir_prefix, namespace)
```

Comment:
This part keeps reappearing in the code. IMO if TF cluster dir prefix and namespace are used only for calculating TF folder, you can make it a parameter and pass around. This will reduce the number of parameters as a bonus.

Reply:
fixed